.github/workflows/release-draft-binaries.yml

```diff
@@ -195,214 +194,209 @@ jobs:
+          asset_path: ${{ env.ASSET_OCKAM_CLI }}.sig
+          asset_name: ${{ env.ASSET_OCKAM_CLI }}.sig
+          asset_content_type: application/octet-stream
+
+      - name: Upload CLI release archive to AWS
+        uses: ./.github/actions/aws_upload
+        with:
+          aws_role: ${{ env.AWS_ROLE }}
+          aws_role_session_name: aws_upload
+          aws_region: ${{ env.AWS_REGION }}
+          bucket_name: ${{ env.BUCKET_NAME }}
+          file_name: ${{ env.ASSET_OCKAM_CLI }}
+          release_version: "v${{ needs.create_release.outputs.version }}"
 
+      - name: Upload CLI Signature to AWS
+        uses: ./.github/actions/aws_upload
+        with:
+          aws_role: ${{ env.AWS_ROLE }}
+          aws_role_session_name: aws_upload
+          aws_region: ${{ env.AWS_REGION }}
+          bucket_name: ${{ env.BUCKET_NAME }}
+          file_name: ${{ env.ASSET_OCKAM_CLI }}.sig
+          release_version: "v${{ needs.create_release.outputs.version }}"
+
+      - name: Upload MacOS App release to GitHub
+        uses: actions/upload-release-asset@ef2adfe8cb8ebfa540930c452c576b3819990faa
+        if: ${{ env.ASSET_OCKAM_APP_DMG }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ needs.create_release.outputs.upload_url }}
+          asset_path: ${{ env.ASSET_OCKAM_APP_DMG }}
+          asset_name: ${{ env.ASSET_OCKAM_APP_DMG }}
+          asset_content_type: application/octet-stream
+
+      - name: Upload MacOS App release Signature to GitHub
+        uses: actions/upload-release-asset@ef2adfe8cb8ebfa540930c452c576b3819990faa
+        if: ${{ env.ASSET_OCKAM_APP_DMG }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ needs.create_release.outputs.upload_url }}
+          asset_path: ${{ env.ASSET_OCKAM_APP_DMG }}.sig
+          asset_name: ${{ env.ASSET_OCKAM_APP_DMG }}.sig
+          asset_content_type: application/octet-stream
+
+      - name: Upload MacOS App release to AWS
+        uses: ./.github/actions/aws_upload
+        if: ${{ env.ASSET_OCKAM_APP_DMG }}
+        with:
+          aws_role: ${{ env.AWS_ROLE }}
+          aws_role_session_name: aws_upload
+          aws_region: ${{ env.AWS_REGION }}
+          bucket_name: ${{ env.BUCKET_NAME }}
+          file_name: ${{ env.ASSET_OCKAM_APP_DMG }}
+          release_version: "v${{ needs.create_release.outputs.version }}"
+
+      - name: Upload MacOS App release Signature to AWS
+        uses: ./.github/actions/aws_upload
+        if: ${{ env.ASSET_OCKAM_APP_DMG }}
+        with:
+          aws_role: ${{ env.AWS_ROLE }}
+          aws_role_session_name: aws_upload
+          aws_region: ${{ env.AWS_REGION }}
+          bucket_name: ${{ env.BUCKET_NAME }}
+          file_name: ${{ env.ASSET_OCKAM_APP_DMG }}.sig
+          release_version: "v${{ needs.create_release.outputs.version }}"
 
   build_elixir_nifs:
```

Comment:
Let's only move the `build_elixir_nifs` job to Nix for this PR. Please revert all changes in the other Jobs

Reply:
I think the issue is with the formatter I'm using...